k-distribution/include/kframework/builtin/domains.md

```diff
@@ -1093,7 +1093,7 @@ You can:
 * Check inequality of two boolean values.
 
 Note that only `andThenBool` and `orElseBool` are short-circuiting. `andBool`
-and `orBool` may be short-circuited in concrete backends, but in symbolic 
+and `orBool` may be short-circuited in concrete backends, but in symbolic
 ackends, both arguments will be evaluated.
```

Comment:
As a side note, this statement about shortcut evaluation is probably not correct, given we have rules/simplifications like `rule _:Bool andBool false => false`  that focus only on the 2nd argument.